crypt-bcrypt.c

```diff
@@ -1028,11 +1029,14 @@ crypt_bcrypt_x_rn (const char *phrase, size_t ARG_UNUSED (phr_size),
 }
 
 void
-gensalt_bcrypt_x_rn (unsigned long count,
-                     const uint8_t *rbytes, size_t nrbytes,
-                     uint8_t *output, size_t o_size)
+gensalt_bcrypt_x_rn (ARG_UNUSED(unsigned long count),
+                     ARG_UNUSED(const uint8_t *rbytes),
+                     ARG_UNUSED(size_t nrbytes),
+                     ARG_UNUSED(uint8_t *output),
+                     ARG_UNUSED(size_t o_size))
 {
-  BF_gensalt ('x', count, rbytes, nrbytes, output, o_size);
+  errno = EINVAL;
+  return;
```

Comment:
Do we need a `gensalt_bcrypt_x_rn` at all? Ideally, we shouldn't.

Reply:
It is easier to keep this function, since the function is excpeted to be present by all the autogen magic.  Thus I've made it to set `errno` to `EINVAL` and explicitly return.